crates/solidity/outputs/cargo/crate/src/backend/built_ins.rs

```diff
@@ -327,6 +345,14 @@ impl<'a> BuiltInsResolver<'a> {
             "stop" => Some(BuiltIn::YulStop),
             "sub" => Some(BuiltIn::YulSub),
             "suicide" if self.language_version < VERSION_0_5_0 => Some(BuiltIn::YulSuicide),
+            "swap1" | "swap2" | "swap3" | "swap4" | "swap5" | "swap6" | "swap7" | "swap8"
```

Comment:
I wonder if these can be generated from the language definition?

Reply:
On the other hand, I see a few here (like `swap11`) are missing from there.

Reply:
I think it should be possible to generate part of this file automatically from the language definition. We may even want to improve on that definition, since it's mostly tailored to what we could represent with stack graphs and some information is lost or not precisely typed.

Reply:
I investigated implementing this and for the moment I think it's not worth it. Given how we have the built-ins declared in the language definition, we would need to hand write many special cases in the generation anyway, and there are a couple of missing built-ins in the definition such as `swap*` or `dup*` (I don't recall if we explicitly ignored them or we simply forgot to include them). Once we deprecate the stack graph bindings, we may want to revise the language definition and adapt it to be able to efficiently generate this module.